src/Player/Player.cs

```diff
@@ -1,11 +1,23 @@
 public class Player
 {
-  public string playerName = "Stepper";
-
-  public int playerHealth = 10;
-  public int playerStepLength = 1;
+  string playerName = "Stepper";
+  int playerHealth = 10;
+  int playerStepLength = 1;
```

Comment:
The intention of this property can be more straight forward.

```suggestion
  int stepModificator = 1;
```

Reply:
Ähm, resolve with other suggestion Dann machste einfach